src/libImaging/FliDecode.c

```diff
@@ -232,7 +232,8 @@ ImagingFliDecode(Imaging im, ImagingCodecState state, UINT8 *buf, Py_ssize_t byt
                 /* Note, have to check Data + size, not just ptr + size) */
                 if (data + (state->xsize * state->ysize) > ptr + bytes) {
                     /* not enough data for frame */
-                    /* UNDONE Unclear that we're actually going to leave the buffer at the right place. */
+                    /* UNDONE Unclear that we're actually going to leave the buffer at
+                     * the right place. */
```

Comment:
```suggestion
                    /* UNDONE Unclear that we're actually going to leave the buffer */
                    /* at the right place. */
```

Reply:
It looks a little odd to mix the comment style with the preceding line, let's use `//` style for the `/* not enough data for frame */` as well.

PEP 7 uses C++-style `//` https://peps.python.org/pep-0007/#c-dialect but we don't need to go through and change them everywhere.

Reply:
Sure - https://github.com/hugovk/Pillow/pull/119